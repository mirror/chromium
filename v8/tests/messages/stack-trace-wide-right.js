function f() {
  "This is a very long string which have to be there to make this line which raises an exception to be more that 78 characters long"; xxx();
}

f();
