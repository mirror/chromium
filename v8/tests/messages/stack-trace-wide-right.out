stack-trace-wide-right.js:2: Uncaught ReferenceError: xxx is not defined
his line which raises an exception to be more that 78 characters long"; xxx();
                                                                        ^
